docker-compose.yml

```diff
@@ -55,3 +74,7 @@ services:
 
 volumes:
     postgres-volume:
+
+networks:
+    grobid:
```

Comment:
Since `elife-xpub` doesn't care much about Grobid, in the context of this configuration this network could be called `sciencebeam-internal` or similar, exemplifying that it's where components of Sciencebeam should stay hidden